### ISSUE TITLE
Adding Delete button

### ISSUE THREAD

**Issue by szabozoltan69** [eledwinn/vue-editable-grid#17]
It would be fine to give one more option to the 'keydown' event listener, a Delete possibility, via
https://github.com/eledwinn/vue-editable-grid/blob/master/src/components/VueEditableGrid.vue#L142

``` 
     } else if (key === 'Delete') {
        const deleteMe = this.getCell()
        deleteMe.rowData[deleteMe.colData.field] = ''
```

**Comment by eledwinn** [comment]
Hi @szabozoltan69, thank you for your improvement proposal, this feature has been published in version 0.3.16 🚀.

**Comment by szabozoltan69** [comment]
Thank you so much!
